codegen/bin/codegen-ios

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var path = require('path');
+var mkdirp = require('mkdirp');
+var semver = require('semver');
+var optimist = require('optimist');
+
+// var generator = require(loopback-sdk-ios-codegen);
+var generator = require('../lib/objc-codegen');
+
+var argv = optimist
+  .usage('Generate iOS Objective-C client code for your LoopBack application.' +
+    '\nUsage:' +
+    '\n    $0 [options] path-to-app ios-client-dir-path' +
+    '\n' +
+    '\nExample:' +
+    '\n    $0 -p XX server/app.js client/ios/gen-src')
+  .describe('p', 'The prefix for generated iOS object names.')
+  .default('p', '')
+  .alias({ p: 'model-name-prefix' })
+  .demand(2)
+  .argv;
+
+var appFile = path.resolve(argv._[0]);
+var clientDir = path.resolve(argv._[1]);
+var modelPrefix = argv['model-name-prefix'];
+
+console.error('Loading LoopBack app %j', appFile);
+var app = require(appFile);
+// assertLoopBackVersion(appFile);
```

Comment:
If you don't need to assert loopback version, then it's better to remove this code altogether.